infer/src/base/Process.ml

```diff
@@ -36,6 +36,7 @@ let create_process_and_wait ~prog ~args =
 
 
 let pipeline ~producer_prog ~producer_args ~consumer_prog ~consumer_args =
+  let open Core in (* fork *)
```

Comment:
Maybe if `Core_kernel` is the new god then we want to be more cautious about `open`ing `Core`? We could just use `let module Unix = Core.Unix in` here.

Reply:
`Process` and `ProcessPool` are the two files on which I'm a bit stuck, as it is highly non-trivial (I believe) to rewrite them not to use Core. Any suggestion would be appreciated. In particular, I think I remember @mbouaziz mentioning that those files were not needed for the Java backend (or maybe I misunderstood). If so, one possibility would be to isolate a reduced part of Infer in which processes are not used, perhaps available through a config switch?

Reply:
They are used to parallelise the analysis. One option could be to have a configure switch to enable/disable them and forego parallelism on Windows for the time being. This involves moving these files to their own directory and then play the same trick as for clang/ vs clang_stubs/ and java/ vs java_stubs/: if the switch is true then pick one if not the other, but both directories contain the same toplevel interface to the outside world (the _stubs one being just a dummy implementation–in that case a sequential one).

Longer term (hopefully just a few months), we plan to work on using the Hack infrastructure for parallelism instead of this ad-hoc "ParMap"-like implementation, which I believe is compatible with Windows.

Reply:
The non-parallel version of `ProcessPool` is in f395c1bae7f5345f1fe820b421f06b33738e94e6. The build trick machinery I used is a bit different, but it works. Your alternative was not playing nice for a single file in the middle of a directory/library.